provisioning/device/src/tpm_registration.ts

```diff
@@ -26,6 +28,7 @@ export class TpmRegistration extends EventEmitter implements RegistrationClient
     this._idScope = idScope;
     this._pollingStateMachine = new PollingStateMachine(transport);
 
+
```

Comment:
nit: unnecessary whitespace

Reply:
Done.

---
In reply to: [272699366](https://github.com/Azure/azure-iot-sdk-node/pull/524#discussion_r272699366) [](ancestors = 272699366)